### DISCUSSION TITLE
Getting Started with MOJO as a noob

### DISCUSSION THREAD

**Discussion by ALI3Nass** [modular/modular#79] [Q&A] [answered]
Can I start learning MOJO with little much knowledge of python or as a beginner to programming.

**Comment by ysyecust** [comment, marked as answer]
A little hard, if you have tested the mojo jupyter-notebook, you'll know that some computer knowledge is required to get the most out of it.

**Comment by hebbaraditya** [comment]
its little bit hard because some of the things are related to C things, and compiler technologies which is great and with practice we can do that i guess,  I am a beginner here as well!!

**Comment by lattner** [comment]
agreed with the above.  Mojo is brand new, I'd recommend starting by learning Python first.  If you are an experienced python programmer, please sign up for the notebooks and work through the examples
